azure/iac/storageaccounts.rego

```diff
@@ -205,7 +205,7 @@ blobService = false {
     azure_issue["blobService"]
 }
 
-blobService {
+blobService = false {
```

Comment:
@shahinshirvani  if attribute is missing, the encryption feature get enabled by default. previous rule was correct. no need to change. see my comment at the top of the rule.

Reply:
@rezoan correct, but still need some changes